scripts/archlinux_install.sh

```diff
@@ -0,0 +1,280 @@
+# trap errors 
+set -e
+
+PATH_PREFIX=/usr/local
+CURRENT_DIR=$(pwd)
+
+if [ "$USER" == "root" ] ; then
+ HOST_SOURCE_BRANCH=$PATH_PREFIX/src/opencog  
+ HOST_BUILD_DIR=/tmp/opencog_build
+else
+ HOST_SOURCE_BRANCH=$CURRENT_DIR/opencog/src  
+ HOST_BUILD_DIR=$CURRENT_DIR/opencog/build
+fi
+
+SLEEP_TIME=0
+
+DEFAULT_PACKAGE_TYPE=local		
+DEFAULT_ADD_REPOSITORIES=true
+DEFAULT_INSTALL_DEPENDENCIES=true	
+DEFAULT_UPDATE_OPENCOG=true		
+#DEFAULT_BUILD_OPENCOG=true		
+#DEFAULT_TEST_OPENCOG=true		
+
+
+SELF_NAME=$(basename $0)
+TOOL_NAME=octool
+
+# get processor count
+PROCESSORS=$(grep "^processor" /proc/cpuinfo | wc -l)
+
+# make job count equal to processor count
+MAKE_JOBS=$(($PROCESSORS+0))
+
+
+LIVE_SOURCE_BRANCH=$HOST_SOURCE_BRANCH
+LIVE_BUILD_DIR=$HOST_BUILD_DIR
+
+VERBOSE="-v"				#for mount, umount, rm, etc.
+QUIET="" 				#for apt-get
+
+LIVE_DESKTOP_SOURCE="OpenCog Source Code"
+
+
+PACKAGES_TOOLS="	
+		squashfs-tools \
+		libisoburn \
+		libisofs \
+		"
+
+PACKAGES_FETCH="	
+		git \
+			"
+		#bzr-rewrite \
+
+PACKAGES_ADMIN="epiphany-browser \
+		"
+
+PACKAGES_BUILD="
+		base-devel	
+		cmake \
+		cxxtest \
+		rlwrap \
+		guile \
+		icu
+		bzip2 \
+		cython \
+		python \
+		python-pyzmq \
+		python-matplotlib \
+		python-nose \
+		python-numpy \
+		python-scipy \
+		python-mock \
+		python-simplejson \
+		python-pyparsing \
+		python-dateutil \
+		python-six \
+		python-tornado \
+		boost-libs \
+                zeromq \
+                intel-tbb \
+                binutils \
+                gsl \
+                unixodbc \
+                xerces-c \
+                libutil-linux \
+                protobuf \
+                protobuf-c \
+                gnurl \
+                curl \
+                expat \
+
+		sdl_gfx \
+		openssl \
+		tcl \
+		tcsh \
+		freetype2 \
+		blas \
+		gcc-fortran \
+		"
+		#liblua5.1-0-dev \
+		#libxmlrpc-c3-dev \
+		#python-flask \
+		#python-flask-restful \
+
+PACKAGES_RUNTIME="	
+		unixodbc \
+		psqlodbc \
+		libpqxx \
+		"
+
+PACKAGES_DOC="		
```

Comment:
@misgeatgit  yes it is. @yimesgen use https://github.com/opencog/opencog/blob/master/opencog/python/requirements.txt for installing python dependencies